packages/rich-text/src/test/unregister-format-type.js

```diff
@@ -0,0 +1,47 @@
+/**
+ * External dependencies
+ */
+import { noop } from 'lodash';
+
+/**
+ * Internal dependencies
+ */
+import { unregisterFormatType } from '../unregister-format-type';
+import { registerFormatType } from '../register-format-type';
+import { getFormatTypes } from '../get-format-types';
+
+describe( 'unregisterFormatType', () => {
+	const defaultFormatSettings = { edit: noop, title: 'format title' };
+
+	// Initialize format store.
+	require( '../store' );
+
+	afterEach( () => {
+		getFormatTypes().forEach( ( format ) => {
+			unregisterFormatType( format.name );
+		} );
+	} );
+
+	it( 'should fail if the format is not registered', () => {
+		const oldFormat = unregisterFormatType( 'core/test-format' );
+		expect( console ).toHaveErroredWith( 'Format core/test-format is not registered.' );
+		expect( oldFormat ).toBeUndefined();
+	} );
+
+	it( 'should unregister existing formats', () => {
+		registerFormatType( 'core/test-format', defaultFormatSettings );
+		expect( getFormatTypes() ).toEqual( [
```

Comment:
It might read better when `getFormatType` is used here as `core/test-format'` is the only thing we care about here..